### DISCUSSION TITLE
posts/2023/47c4b777/

### DISCUSSION THREAD

**Discussion by giscus** [GZTimeWalker/blog-comments#18] [Announcements]
# posts/2023/47c4b777/

一手带起的 CTF 队伍，到如今令众多队伍骨干备受折磨，发生了什么？

https://blog.gzti.me/posts/2023/47c4b777/

**Comment by GrakePch** [comment]
🫶

**Reply by GrakePch** [reply]
emoji怎么没有比较人情味的拥抱表情（恼

**Comment by xxuuan** [comment]
加油吧 兄弟

**Comment by MUZIHIT** [comment]
作为一个刚入行ctf一年多的新人，也算是我们学校ctf组织建立的最早一批人，也在建立社团、打比赛，虽然大家水平都没到拿奖金的程度，甚至很多比赛都不能打到线下赛，但是大家还是都很开心，看着周围的战友兴奋地讨论如何拿到flag和对技术的热爱，真的让我感到纯正的快乐。看到博主所写的字句，深有感触，也会想周围这份对ctf的热爱还能维持多久，可能最后也会有很多陪伴自己打出名号的战友抛弃了ctf最原始的热爱，但也希望始终有一部分人可以坚守在那片净土，相信热爱CTF的人，永远不会孤身一人  ：wq

**Comment by syuy** [comment]
加油
